### PR TITLE
[Docs] Add a blurb about how Alerts work

### PR DESCRIPTION
**Pull request by zengk95**
TSIA

### CODE REVIEW

File: torchci/README.md [hunk 1]
```diff
@@ -87,10 +87,21 @@ console.
 4. Update `rockset/prodVersion.json` with the new version of the lambda.
 
 
+### Alerts
+
+The scripts/check_alerts.py queries HUD, filters out pending jobs, and then checks to see if there are 2 consecutive
+SHAs that have the same failing job. If it does, it will either create a new Github Issue or update the existing
+Github Issue.
+
+Once the issue is created, internal webhooks creates a task out of the issue. Then a [butterfly bot
```
Comment (ZainRizvi) [inline torchci/README.md:96]: typo 

```suggestion
Once the issue is created, internal webhooks create a task out of the issue. Then a [butterfly bot
```

**Review by ZainRizvi — APPROVED** [review]
This is really helpful, thanks!

File: torchci/README.md [hunk 2]
```diff
@@ -87,10 +87,21 @@ console.
...
+
 ### Modifying Deployment Settings
 
 If you ever need to modify the deployment settings like the oauth callbacks, domain names, there's a few places that you need to change these settings in. Here's a list:
 1. [DNS Registry/Certificates](https://fb.workplace.com/groups/osssupport) (Contact the the OSS team)
 2. [Environment Variables](https://vercel.com/torchci/torchci/settings/environment-variables)
 3. [OAuth Project](https://github.com/settings/applications/1973779) / [OAuth Project Local](https://github.com/settings/applications/1976306)
-4. [Domain Management](https://vercel.com/torchci/torchci/settings/domains)
\ No newline at end of file
+4. [Domain Management](https://vercel.com/torchci/torchci/settings/domains)
```
Comment (huydhn) [inline torchci/README.md:106]:
```suggestion
4. [Domain Management](https://vercel.com/fbopensource/torchci/settings/domains)
```

File: torchci/README.md [hunk 3]
```diff
@@ -87,10 +87,21 @@ console.
...
+
+Once the issue is created, internal webhooks creates a task out of the issue. Then a [butterfly bot
+rule](https://www.internalfb.com/butterfly/rule/2024866984357962) will trigger to assign the task to 
+the oncall and update it to a UBN so that it alerts the current oncall.
+
 ### Modifying Deployment Settings
 
 If you ever need to modify the deployment settings like the oauth callbacks, domain names, there's a few places that you need to change these settings in. Here's a list:
 1. [DNS Registry/Certificates](https://fb.workplace.com/groups/osssupport) (Contact the the OSS team)
 2. [Environment Variables](https://vercel.com/torchci/torchci/settings/environment-variables)
```
Comment (huydhn) [inline torchci/README.md:104]:
```suggestion
2. [Environment Variables](https://vercel.com/fbopensource/torchci/settings/environment-variables)
```

Comment (huydhn) [reply torchci/README.md:104]: Ignore these if they are fixed already (in other PRs)

File: torchci/README.md [hunk 4]
```diff
@@ -87,10 +87,21 @@ console.
...
 
 
+### Alerts
+
+The scripts/check_alerts.py queries HUD, filters out pending jobs, and then checks to see if there are 2 consecutive
+SHAs that have the same failing job. If it does, it will either create a new Github Issue or update the existing
+Github Issue.
+
+Once the issue is created, internal webhooks creates a task out of the issue. Then a [butterfly bot
+rule](https://www.internalfb.com/butterfly/rule/2024866984357962) will trigger to assign the task to
```
Comment (huydhn) [inline torchci/README.md:97]: Just curious: is there any policy preventing us putting internal links here (or in other OSS repos for that matter)?

**Review by huydhn — APPROVED** [review]
Some minor tweaks are needed, but LGTM otherwise!

File: torchci/README.md (hunk 4 above)
Comment (zengk95) [reply torchci/README.md:97]: No it's fine to put internal links here. You can't access internal unless you're on VPN/certain certs so it should be fine. External people will just have to depend on us to edit butterfly bot stuff.

**Pull request merged**